nautilus_trader/adapters/interactive_brokers/parsing/instruments.py

```diff
@@ -59,6 +60,22 @@ def sec_type_to_asset_class(sec_type: str):
     return asset_class_from_str(mapping.get(sec_type, sec_type))
 
 
+def get_contract_details_tick_scheme_name(details: ContractDetails) -> str:
+    exchange_to_market_rules = dict(
+        zip(details.validExchanges.split(","), details.marketRuleIds.split(",")),
+    )
+    unique_market_rules = sorted(set(exchange_to_market_rules.values()))
```

Comment:
If `SMART` present it will be at index so we don't need to sort it. In either case we can assign Index 0 as default to the Instrument.
I think what we can do is have an optional parameters (exchange: str = None). If that's none it returns default (Index 0). If the value is present for `exchange` it will return the schema for that exchange or raise error if not found. So then the same function `get_contract_details_tick_scheme_name` can be used within Strategy as well in case placing order to any other exchange.